### ISSUE TITLE
iosSimulatorArm64 support

### ISSUE THREAD

**Issue by luca992** [icerockdev/moko-web3#114]
So I already implemented this on my fork by creating a separate branch: [ionspinbignum](https://github.com/luca992/moko-web3/tree/ionspinbignum) 

It is using ionspin's [bignum library](https://github.com/ionspin/kotlin-multiplatform-bignum) which supports arm64 targets already instead of soywiz's [kbignum](https://github.com/korlibs/kbignum) as the big number library. kbignum would need to put a new release out to support arm64.

In addition, to me, it seems like ionspin's library is more actively maintained and has more features. Would switching to using ionspin's big number library be a possibility to support arm64 targets?

**Comment by y9san9** [comment]
Yes, I saw that kbignum doesn't provide a proper way to use it with arm64, but it is already used in every of our multiplatform project, so we are locked with this :(

By the way, you can write to the creator directly in Kotlin Slack channel
